### ISSUE TITLE
Location of custom visual source code in GitHub

### ISSUE THREAD

**Issue by cborovac** [MicrosoftDocs/powerbi-docs#566]
Hi - Can you let me know if the source code is also available for PowerBI's standard visuals? This would be useful to use as a template when I am developing custom visuals. Can you please also provide a link to the custom visual source code referenced in this article located within GitHub?
Thanks. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5c4bae6e-72e1-e41c-4419-48a56423e81e
* Version Independent ID: 6d864495-b578-2d28-169d-edb5409de943
* Content: [Developing a Power BI Custom Visual - Power BI](https://docs.microsoft.com/en-us/power-bi/developer/custom-visual-develop-tutorial)
* Content Source: [powerbi-docs/developer/custom-visual-develop-tutorial.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/live/powerbi-docs/developer/custom-visual-develop-tutorial.md)
* Service: **powerbi**
* GitHub Login: @markingmyname
* Microsoft Alias: **maghan**

**Comment by mihart** [comment]
Hello @cborovac 
Thank you for contacting us.  I've forwarded your question to the author of the article, who will respond soon.
Best regards,
Microsoft DOCS International Team

**Comment by markingmyname** [comment]
Hi @cborovac, thank you for your comment.  We cannot release the source code for the standard visuals, but there is a link in the article that references the custom visual source code.

https://github.com/Microsoft/PowerBI-visuals-circlecard

Hope this help.

Thank you.

**Comment by markingmyname** [comment]
We will now proceed to close this thread. If there are further questions regarding this matter, please tag me in your reply. We will gladly continue the discussion and we will reopen the issue.

**Comment by JeyanthVictory** [comment]
@markingmyname , when I tried to run the project in the github link you provided, it throws error.
![image](https://user-images.githubusercontent.com/36911889/65485065-6455ae00-debe-11e9-974b-8c14ef88e73f.png)

I manually installed the required peer dependencies and did `npm start` again. It throws

![image](https://user-images.githubusercontent.com/36911889/65485122-8ea76b80-debe-11e9-8d1f-d3478fc9354c.png)

Kindly help me to get rid of this.
